test_files/export_import/test.ts

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
nit: I usually name these `export_import/export_import.ts` so that we don't end up with a million files called `test.ts`. The repetition sucks too, not sure if it's substantially better...